Docker_files/scripts/mg_external.sh

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env bash
```

Comment:
Not sure if Docker_files is right folder. I think we should move it or create other one as it's not related to any docker file or so.

Reply:
Moved to `scripts/bash/mg_external.sh`